<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1236</width>
    <height>712</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1090</width>
    <height>636</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>制作器</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="hello_paper">
       <attribute name="title">
        <string>欢迎</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="hello_html">
          <property name="styleSheet">
           <string notr="true">border-radius:10px;
border: 2px groove gray;
padding: 1px 18px 1px 3px;
background-color: rgba(255, 255, 255, 0);</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="posedit_paper">
       <attribute name="title">
        <string>位置选定</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="7" column="2" colspan="6">
         <widget class="QPushButton" name="getimg_bt">
          <property name="text">
           <string>打开图片</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" rowspan="5" colspan="6">
         <widget class="QGraphicsView" name="view">
          <property name="minimumSize">
           <size>
            <width>700</width>
            <height>500</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="y_pos">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Y:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QPushButton" name="delval_bt">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>删除文字</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QListWidget" name="vallist_li">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>500</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QRadioButton" name="yuan_mode">
          <property name="text">
           <string>打开时使用原照片分辨率</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QPushButton" name="setfont_bt">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>字体</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="addval_bt">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>添加文字</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="x_pos">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>X:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="Line" name="line">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="5" column="6">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>放大</string>
          </property>
         </widget>
        </item>
        <item row="5" column="5">
         <widget class="QSlider" name="SceneSuoFang">
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="setcolor_bt">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>字体颜色</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>缩小</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QPushButton" name="reset_bt">
          <property name="text">
           <string>复位</string>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QRadioButton" name="nomarl_mode">
          <property name="text">
           <string>打开时使用压缩分辨率</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="6" column="5">
         <widget class="QRadioButton" name="ek_mode">
          <property name="text">
           <string>打开时固定8k分辨率</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="batchmake_paper">
       <attribute name="title">
        <string>批量制作</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" rowspan="3">
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>319</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>419</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>文字列表</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>选中项:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="selectional">
             <property name="text">
              <string>null</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QListWidget" name="vals">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>400</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>导入名单</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="importList_bt">
             <property name="text">
              <string>导入</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="start_make_bt">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>200</height>
              </size>
             </property>
             <property name="text">
              <string>开始制作</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="2">
            <widget class="QTextEdit" name="input_list"/>
           </item>
           <item row="1" column="1">
            <widget class="QRadioButton" name="isten">
             <property name="text">
              <string>使用10进制量</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>名单预览</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QTableWidget" name="namelist">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="columnCount">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1236</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
